Don't print args/kwargs on fake tensor error.

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #102620

Under certain obscure circumstances (notably while I was debugging
https://github.com/pytorch/pytorch/issues/101160 ) this printing
causes PyTorch to hang.  I tried a bit to root cause the issue but
couldn't work it out, sending this in to unblock for now.

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

ezyang committed:
Don't print args/kwargs on fake tensor error.

Under certain obscure circumstances (notably while I was debugging
https://github.com/pytorch/pytorch/issues/101160 ) this printing
causes PyTorch to hang.  I tried a bit to root cause the issue but
couldn't work it out, sending this in to unblock for now.

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

[ghstack-poisoned]

bdhirsh commented on torch/_subclasses/fake_tensor.py:
```diff
@@ -1417,12 +1417,12 @@ def validate(x):
                 if not self.allow_non_fake_inputs:
                     raise Exception(
                         f"Please convert all Tensors to FakeTensors first or instantiate FakeTensorMode "
-                        f"with 'allow_non_fake_inputs'. Found in {func}(*{args}, **{kwargs}) "
```
Maybe just keep the `func`? Seems like the most important piece of info

ezyang replied:
sure

ezyang committed:
Update on "Don't print args/kwargs on fake tensor error."


Under certain obscure circumstances (notably while I was debugging
https://github.com/pytorch/pytorch/issues/101160 ) this printing
causes PyTorch to hang.  I tried a bit to root cause the issue but
couldn't work it out, sending this in to unblock for now.

Signed-off-by: Edward Z. Yang <ezyangmeta.com>

[ghstack-poisoned]
